config/src/test/java/tech/pegasys/pantheon/config/IbftConfigOptionsTest.java

```diff
@@ -26,6 +26,7 @@
   private static final int EXPECTED_DEFAULT_EPOCH_LENGTH = 30_000;
   private static final int EXPECTED_DEFAULT_BLOCK_PERIOD = 1;
   private static final int EXPECTED_DEFAULT_REQUEST_TIMEOUT = 1;
+  private static final int EXPECTED_DEFAULT_MESSAGE_BUFFER_SIZE = 10_000;
```

Comment:
Are these oddly named? I.e. what does "EXPECTED_" indicate?

Reply:
They are bit oddly named I guess. The "EXPECTED_" indicates this expected values for the assertion. These values are used to assert the default values are correct in the tests. It's existing test code so I just followed the existing pattern.